its/tests/src/test/java/its/SonarCloudTests.java

```diff
@@ -195,7 +193,7 @@ static void prepare() throws Exception {
   static void cleanup() throws Exception {
     adminWsClient.userTokens()
       .revoke(new RevokeRequest().setName(TOKEN_NAME));
```

Comment:
I think you could remove this? And also the creation of the token, let's try to keep a single one